lib/paths.cf

```diff
@@ -138,6 +138,7 @@ bundle common paths
       "path[timedatectl]"    string => "/usr/bin/timedatectl";
 
     linux::
+      "path[date]"          string => "/usr/bin/date";
```

Comment:
alpine which uses busybox has `/bin/date` but of course you can install a proper `date` in the right place most likely.